_docs_integrate/establish-relationships.md

```diff
@@ -81,6 +81,20 @@ If you have successfully created the [RelationshipTemplate]({% link _docs_integr
 
 {% include copy-notice description="Save the `id` of the RelationshipTemplate so that you can refer to it and make it available to other Identities later. For the same reason, save the value of the property `truncatedReference`." %}
 
+### Personalize the RelationshipTemplate
```

Comment:
Please put this section before the `### Successfully Created RelationshipTemplate` section and after the `### Input for Creating a RelationshipTemplate` section.